backend/src/models/base/BaseModel.js

```
@@ -0,0 +1,236 @@
+         return [];
+      }
+      return dbRows.map((row) => this.fromDb(row));
+   }
+
+   /**
+    * Create model from user input (req.body)
+    * Override in child classes
+    */
+   static fromInput(inputData) {
+      throw new Error(
+         `fromInput method must be implemented in ${this.constructor.name}`
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)